### PR TITLE
Reverts a Skyrat laser pointer chase change partially.

### PR DESCRIPTION
**Pull request by pixelkitty286**
## About The Pull Request
Reverts https://github.com/Skyrat-SS13/Skyrat-tg/pull/25588 partially to allow for laser chasing once more

## Why It's Good For The Game

I quite found the stories of people chasing laser lights quite funny, and as a cat player my self I feel that change that SkyRat made was quite poor sighted and would like to see it gone on bubber.

## Proof Of Testing

image

## Changelog

:cl:
del: looking at lasers only
code: commented out code from skyrat
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/laserpointer.dm [hunk 1]
```diff
@@ -275,14 +275,12 @@
 			continue
 		if(target_felinid.body_position == STANDING_UP)
 			target_felinid.setDir(get_dir(target_felinid, targloc)) // kitty always looks at the light
-			//SKYRAT EDIT REMOVAL BEGIN (removes forced felinid movement from laserpointers, also fixes the longstanding windoor negation glitch)
-			/* if(prob(effectchance * diode.rating))
+			if(prob(effectchance * diode.rating))
```
Comment (nikothedude) [inline code/game/objects/items/devices/laserpointer.dm:278]: dont you hvae to do a bubber edit instead of just reverting it

Comment (projectkepler-ru) [reply code/game/objects/items/devices/laserpointer.dm:278]: correct

**Review by Majkl-J — APPROVED** [review]
I feel like this can be removed upstream

**Comment by pixelkitty286** [comment]
> I feel like this can be removed upstream

Fatha i am scared of skyrat

**Comment by projectkepler-ru** [comment]
> > I feel like this can be removed upstream
> 
> Fatha i am scared of skyrat

be not afraid, for we will be your light( we're maintainers on skyrat)

**Comment by aKromatopzia** [comment]
half-sad there's no real system for species specific quirks (hydrophillic, masquerade)

**Comment by pixelkitty286** [comment]
> half-sad there's no real system for species specific quirks (hydrophillic, masquerade)

off topic

**Comment by Majkl-J** [comment]
I did this upstream :3

**Pull request closed without merging**